src/auth/discord-auth.service.ts

```diff
@@ -40,10 +40,12 @@ export class DiscordAuthService implements IAuthProvider {
                 "[discord-auth.service]: Cannot get email from discord to create a new Chingu account",
             );
 
+        const existingUser = await this.findUserByEmails([user.email]);
```

Comment:
Is it possible that this could return null?

Reply:
Yes it can https://www.prisma.io/docs/orm/reference/prisma-client-reference#findunique